nerdlandbot/commands/GuildData.py

```diff
@@ -263,7 +267,45 @@ async def update_notification_audit(self, list_name: str) -> bool:
         await self.save()
 
         return True
+
+    async def update_church_channel(self, church: str) -> bool:
+        """
+        Updates the kerk_channel
+        :param kerk: the channel that's been set
+        :return: True if updated and saved, False if it's the same
+        """
+        church = church.strip("<#")
+        church = int(church.strip(">"))
+        if church != self.church_channel:
```

Comment:
Why can't you just update the channel, even if you are updating it to the same value?

Reply:
Honestly, I don't know. It's been a while since I've written this code.